SaltSquareIO/sports-app-react-fe
# Add logo and change project name

[CLICKUP_TICKET_ID](https://app.clickup.com/t/86c0m6unb)

## Summary of change

This PR introduces the following changes: 
- Updated the project name from **sports-app-react-fe** to **profile-app-react**.
- Changed any mentions of **sports app** to **profile app**.
- Replaced the default React logo icon with our own.  

## QA Notes

1. Launch the app and verify that the title in the browser tab shows Profile App when navigating through the app.
2. Ensure that the default React logo icon has been replaced with the new custom one. 
3. Ensure that all mentions of "Sports App" have been updated to "Profile app" across the codebase.
